### PR TITLE
Lets you flip the Captain and hop's hats

### PR DESCRIPTION
**Pull request by Supermichael777**
:cl: Supermichael777
rcsadd: The Captain and Hop can now flip their hats. This is pointless.
/:cl:

### CODE REVIEW

**Comment by optimumtact** [comment]
Why?

**Comment by Supermichael777** [comment]
Why not?

**Comment by optimumtact** [comment]
Eh I suppose

File: code/modules/clothing/head/jobs.dm [hunk 1]
```diff
@@ -19,26 +19,28 @@
 	return(FIRELOSS)
 
 //Captain
-/obj/item/clothing/head/caphat
+/obj/item/clothing/head/soft/caphat
 	name = "captain's hat"
 	desc = "It's good being the king."
-	icon_state = "captain"
+	icon_state = "captainsoft"
+	item_color = "captian"
```
Comment (ike709) [inline code/modules/clothing/head/jobs.dm]: pretty sure this is a typo?

Comment (Supermichael777) [inline code/modules/clothing/head/jobs.dm]: nope look at softcaps shitty code
color is how it updates item sprites insted of doing something sane like having a flipped sprite var and reverting using inital

Comment (ExcessiveUseOfCobblestone) [inline code/modules/clothing/head/jobs.dm]: so item color "Captian" and not "Captain" is intended?

Comment (Supermichael777) [inline code/modules/clothing/head/jobs.dm]: GAH FUCK

**Comment by Cheridan** [comment]
Sprites?

**Comment by Cheridan** [comment]
i don't understand

**Pull request closed without merging**